src/solver/optimisation/post_process_commands.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../simulation/base_post_process.h"
+
+namespace Antares::Solver::Simulation
+{
+
+class DispatchableMarginPostProcessCmd : public basePostProcessCommand
+{
+public:
+    DispatchableMarginPostProcessCmd(PROBLEME_HEBDO* problemeHebdo,
+                                     unsigned int thread_number,
+                                     AreaList& areas);
+    void acquireOptRuntimeData(const struct optRuntimeData& opt_runtime_data) override;
+    void run() override;
```

Comment:
Rename **run()** into **execute()**, to stick with the command design pattern, and it can be more recognizable

Reply:
Done